internal/actor/donor_provided_test.go

```diff
@@ -20,19 +20,105 @@ var address = place.Address{
 }
 
 func TestGenerateHash(t *testing.T) {
-	donor := &DonorProvidedDetails{Attorneys: Attorneys{
-		Attorneys: []Attorney{
-			{DateOfBirth: date.New("2000", "1", "2")},
+	testcases := map[string]struct {
+		donor    *DonorProvidedDetails
+		initial  uint64
+		modified uint64
+	}{
+		"version 0": {
+			donor: &DonorProvidedDetails{
+				HashVersion: 0,
+				Attorneys: Attorneys{
+					Attorneys: []Attorney{
+						{DateOfBirth: date.New("2000", "1", "2")},
+					},
+				},
+			},
+			// DO NOT change these values. If a field has been added/removed you will
+			// need to handle the version gracefully by modifying HashInclude and
+			// adding another testcase.
```

Comment:
of course I don't think we'll actually follow this before we start collecting real data, unless we want to practise